logrecorder-jul/src/main/kotlin/io/github/logrecorder/jul/kotest/JulRecorderKotestExtension.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.logrecorder.jul.kotest
+
+import io.github.logrecorder.api.LogRecord
+import io.github.logrecorder.common.kotest.AbstractLogRecorderKotestExtension
+import io.github.logrecorder.jul.JulLogRecord
+import io.github.logrecorder.jul.JulLogRecorder
+import java.util.logging.LogManager
+import java.util.logging.Logger
+import kotlin.reflect.KClass
+
+/**
+ * Convenience Method to instantiate [JulRecorderKotestExtension]
+ * This extension will record the loggers specified by the parameters save the [LogRecord] into the CoroutineContext
+ * for retrieval by logRecord
+ *
+ * Recording a logger will set its log level to [java.util.logging.Level.FINER] for the duration of the test.
+ * After the test was executed, the log level will be restored to whatever it was before.
+ *
+ * @param classes [KClass] whose name should be used to identify loggers to record.
+ * @param names String names of the [Logger] instances to record
+ * @see JulRecorderKotestExtension
+ * @since 2.4
+ */
+@PublishedApi
+internal fun recordLogs(
+    vararg classes: KClass<*>,
+    names: Array<out String> = emptyArray()
+) = JulRecorderKotestExtension(classes, names)
+
+/**
+ * @see recordLogs
+ * @since 2.4
+ */
+@PublishedApi
+internal fun recordLogs(
```

Comment:
if they are all `internal`, nobody can use them - how do I use the extension?

Reply:
The @PublishedApi makes it usable